### ISSUE TITLE
Cannot export tables from Postgres

### ISSUE THREAD

**Issue by Setitch** [HeidiSQL/HeidiSQL#1123]
**Describe the bug**
When trying to export table with data the message:  ,,Could not find table or view .... Please refresh database tree...''

**To Reproduce**
1. Go to export  (tale tools)
2. Select table (my table name:  guidelines-divisions)
3. Select: Data ...  as insert, or any selection that would export data rows as well

**Expected behavior**
As table exists, and it even finds out how many rows there is - it should properly export the table with data into sql;

**SQL LOG**
```
SELECT * FROM "information_schema"."columns" WHERE TABLE_SCHEMA='public' AND TABLE_NAME='guidelines-divisions' ORDER BY ORDINAL_POSITION;
SELECT a.attname AS column, des.description AS comment FROM pg_attribute AS a, pg_description AS des, pg_class AS pgc WHERE     pgc.oid = a.attrelid     AND des.objoid = pgc.oid     AND pg_table_is_visible(pgc.oid)     AND pgc.relname = 'guidelines-divisions'     AND a.attnum = des.objsubid;
WITH ndx_list AS (    SELECT pg_index.indexrelid, pg_class.oid    FROM pg_index, pg_class    WHERE pg_class.relname = 'guidelines-divisions'    AND pg_class.oid = pg_index.indrelid  ),  ndx_cols AS (    SELECT pg_class.relname, UNNEST(i.indkey) AS col_ndx,    CASE i.indisprimary WHEN true THEN 'PRIMARY' ELSE CASE i.indisunique WHEN true THEN 'UNIQUE' ELSE 'KEY' END END AS CONSTRAINT_TYPE,    pg_class.oid    FROM pg_class    JOIN pg_index i ON (pg_class.oid = i.indexrelid)    JOIN ndx_list ON (pg_class.oid = ndx_list.indexrelid)    WHERE pg_table_is_visible(pg_class.oid)  )SELECT ndx_cols.relname AS CONSTRAINT_NAME, ndx_cols.CONSTRAINT_TYPE, a.attname AS COLUMN_NAME FROM pg_attribute a JOIN ndx_cols ON (a.attnum = ndx_cols.col_ndx) JOIN ndx_list ON (ndx_list.oid = a.attrelid AND ndx_list.indexrelid = ndx_cols.oid);
SELECT   refc.constraint_name,   refc.update_rule,   refc.delete_rule,   kcu.table_name,   STRING_AGG(distinct kcu.column_name, ',') AS columns,   ccu.table_schema AS ref_schema,   ccu.table_name AS ref_table,   STRING_AGG(distinct ccu.column_name, ',') AS ref_columns,   STRING_AGG(distinct kcu.ordinal_position::text, ',') AS ord_position FROM   information_schema.referential_constraints AS refc,   information_schema.key_column_usage AS kcu,   information_schema.constraint_column_usage AS ccu WHERE   refc.constraint_schema = 'public'   AND refc.constraint_name = kcu.constraint_name   AND refc.constraint_schema = kcu.table_schema   AND ccu.constraint_name = refc.constraint_name   AND kcu.table_name = 'guidelines-divisions' GROUP BY   refc.constraint_name,   refc.update_rule,   refc.delete_rule,   kcu.table_name,   ccu.table_schema,   ccu.table_name ORDER BY   ord_position;
SELECT 'public' AS "Database", 'guidelines-divisions' AS "Table", 52 AS "Rows", 0 AS "Duration";
/*!40101 SET @OLD_LOCAL_SQL_MODE=@@SQL_MODE, SQL_MODE='' */;
SELECT * FROM "public"."guidelines-divisions" LIMIT 104857600;
/*!40101 SET SQL_MODE=IFNULL(@OLD_LOCAL_SQL_MODE, '') */;
```
**Environment:**
 - OS: Windows 10
 - HeidiSQL version [11.0.0.5919]
 - Database type and version [Postgres 11.6]

**Comment by ansgarbecker** [comment]
Please update to the latest HeidiSQL build and retry (Help > About > Download build).

**Comment by Setitch** [comment]
![screen](https://user-images.githubusercontent.com/2422608/89985991-27340f00-dc7c-11ea-8d13-4589511f4431.png)
Some thing with newest (nighty) version

**Comment by ansgarbecker** [comment]
Cannot reproduce that with the latest build. Probably my fix for #1124 also fixed this one. Please recheck with the current build.

**Comment by Setitch** [comment]
Confirming this #1124 fixed this issue as well
